explainer.md

```diff
@@ -54,18 +55,18 @@ const image = await clipboardItem.getType('image/png');
 
 ## Pickling Write
 Pickled formats will be written when the site tries to use a format that:
-1. Is included in a `{direct: ['format1', 'format2']}` list of formats for the `ClipboardItem`’s `ClipboardItemOption`. 
+1. Is included in a `{unsanitize: ['format1', 'format2']}` list of formats for the `ClipboardItem`’s `ClipboardItemOption`. 
```

Comment:
Would `unsanitized` be better here? (`unsanitize` may be a misspelling)

Reply:
Yes, sorry it was a typo.